### PR TITLE
Update CondTool/SiStrip scripts to use phase1_2022_realistic GT

### PR DESCRIPTION
**Pull request by yuanchao**
#### PR description:

Following up the migration of Geometry_cff to GeometryDB_cff (https://github.com/cms-sw/cmssw/pull/35278), proper
GlobalTag needs to be set in advance. Updates applied to AlCa/DB related packages to use Run 3 GT.

#### PR validation:

Scripts tested locally.

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Not a backport and no backport expected.

### CODE REVIEW

File: CondTools/SiStrip/test/SiStripFedCablingReader_cfg.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
 from Configuration.AlCa.autoCond import autoCond
-process.GlobalTag.globaltag = autoCond['run2_design']
+process.GlobalTag.globaltag = autoCond['phase1_2022_realistic']
```
Comment (mmusich) [inline CondTools/SiStrip/test/SiStripFedCablingReader_cfg.py:37]: please use the design key not the realistic.

File: CondTools/SiStrip/test/SiStripFedCablingBuilder_cfg.py [hunk 2]
```diff
@@ -36,7 +36,7 @@
 
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
 from Configuration.AlCa.autoCond import autoCond
-process.GlobalTag.globaltag = autoCond['run2_design']
+process.GlobalTag.globaltag = autoCond['phase1_2022_realistic']
```
Comment (mmusich) [inline CondTools/SiStrip/test/SiStripFedCablingBuilder_cfg.py:39]: please use the design key not the realistic.

File: CondTools/SiStrip/test/SiStripDetVOffFakeBuilder_cfg.py [hunk 3]
```diff
@@ -33,7 +33,7 @@
 
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
 from Configuration.AlCa.autoCond import autoCond
-process.GlobalTag.globaltag = autoCond['run2_design']
+process.GlobalTag.globaltag = autoCond['phase1_2022_realistic']
```
Comment (mmusich) [inline CondTools/SiStrip/test/SiStripDetVOffFakeBuilder_cfg.py:36]: please use the design key not the realistic.

**Comment by tvami** [comment]
type trk

**Comment by tvami** [comment]
and please also squash the commits

**Comment by yuanchao** [comment]
Squashed the previous commits

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by tvami** [comment]
+db
 * at one point it would be good to have these scripts as unit test

**Comment by mmusich** [comment]
> at one point it would be good to have these scripts as unit test

They already are:

https://github.com/cms-sw/cmssw/blob/ca284edc807fb14a406219b962fe7d5d0da0641a/CondTools/SiStrip/test/testBuildersReaders.sh#L19

https://github.com/cms-sw/cmssw/blob/ca284edc807fb14a406219b962fe7d5d0da0641a/CondTools/SiStrip/test/testBuildersReaders.sh#L28

**Comment by tvami** [comment]
Perfect

**Comment by qliphy** [comment]
+1

**Pull request merged**